<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NovoRegistroMensal</class>
 <widget class="QDialog" name="NovoRegistroMensal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo Registro Mensal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="3">
    <widget class="QPushButton" name="salvar">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Salvar</string>
     </property>
     <property name="icon">
      <iconset resource="Image.qrc">
       <normaloff>:/icons/img/svg/checkmark.svg</normaloff>:/icons/img/svg/checkmark.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Registro Mensal</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mês</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mes"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ano</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ano"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="cancelar">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Fechar</string>
     </property>
     <property name="icon">
      <iconset resource="Image.qrc">
       <normaloff>:/icons/img/svg/export.svg</normaloff>:/icons/img/svg/export.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Contas</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="adicionarConta">
          <property name="text">
           <string>Adicionar</string>
          </property>
          <property name="icon">
           <iconset resource="Image.qrc">
            <normaloff>:/icons/img/svg/add_image.svg</normaloff>:/icons/img/svg/add_image.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editarConta">
          <property name="text">
           <string>Editar</string>
          </property>
          <property name="icon">
           <iconset resource="Image.qrc">
            <normaloff>:/icons/img/svg/edit_image.svg</normaloff>:/icons/img/svg/edit_image.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removerConta">
          <property name="text">
           <string>Remover</string>
          </property>
          <property name="icon">
           <iconset resource="Image.qrc">
            <normaloff>:/icons/img/svg/remove_image.svg</normaloff>:/icons/img/svg/remove_image.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="contas">
        <column>
         <property name="text">
          <string>Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Descrição</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tipo de Conta</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Valor</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Pagante</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Empréstimos e Outros Valores</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="adicionarEmprestimo">
          <property name="text">
           <string>Adicionar</string>
          </property>
          <property name="icon">
           <iconset resource="Image.qrc">
            <normaloff>:/icons/img/svg/add_image.svg</normaloff>:/icons/img/svg/add_image.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editarEmprestimo">
          <property name="text">
           <string>Editar</string>
          </property>
          <property name="icon">
           <iconset resource="Image.qrc">
            <normaloff>:/icons/img/svg/edit_image.svg</normaloff>:/icons/img/svg/edit_image.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removerEmprestimo">
          <property name="text">
           <string>Remover</string>
          </property>
          <property name="icon">
           <iconset resource="Image.qrc">
            <normaloff>:/icons/img/svg/remove_image.svg</normaloff>:/icons/img/svg/remove_image.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="emprestimos">
        <column>
         <property name="text">
          <string>Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Descrição</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>De</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Para</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Valor</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="gerarRelatorio">
     <property name="text">
      <string>Gerar Relatório</string>
     </property>
     <property name="icon">
      <iconset resource="Image.qrc">
       <normaloff>:/icons/img/svg/neutral_trading.svg</normaloff>:/icons/img/svg/neutral_trading.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections/>
</ui>
